Library/Homebrew/utils/pypi.rb

```diff
@@ -58,17 +50,38 @@ def get_pypi_info(package, version)
   end
 
   # Return true if resources were checked (even if no change).
-  def update_python_resources!(formula, version = nil, print_only: false, silent: false,
-                               ignore_non_pypi_packages: false)
-
-    if !print_only && AUTOMATIC_RESOURCE_UPDATE_BLOCKLIST.include?(formula.full_name)
-      odie "The resources for \"#{formula.name}\" need special attention. Please update them manually."
-      return
+  def update_python_resources!(formula, version: nil, package_name: nil, extra_packages: nil, exclude_packages: nil,
+                               print_only: false, silent: false, ignore_non_pypi_packages: false)
+
+    auto_update_list = formula.tap.audit_exceptions[:automatic_resource_update_list]
```

Comment:
Pedantry: this isn't an `audit_exception` in any meaningful form. I think it's probably still worth migrating to Homebrew/core but with a different name/location.

Reply:
Agreed. I suggested `formula_lists` above but am 100% open to other names.

Also, just want to make sure it's okay to rename that in terms of compatibility and whatnot

Reply:
Maybe another dedicated directory (or file in the root) just for the Python resource stuff?

Reply:
Let's move the discussion to https://github.com/Homebrew/brew/pull/9178 (which I'm repurposing for some refactoring of this logic)